gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/FsBenchmark.java

```diff
@@ -241,10 +246,14 @@ private void benchmarkWrite(
                         output.write(writeBuffer);
                         fileBytesWrite += writeSize;
                         writeCallBytes.accept(writeSize);
+
+                        writeCallTime.add(System.nanoTime() - writeCallStart);
```

Comment:
This can be huge, correct? Not sure if this is the right thing to do since this can interfere with the benchmark and give a wrong result.